Content.Shared/Interaction/Components/ClumsyComponent.cs

```diff
@@ -9,14 +9,21 @@ namespace Content.Shared.Interaction.Components
     [RegisterComponent]
     public sealed partial class ClumsyComponent : Component
     {
-        [DataField("clumsyDamage", required: true)]
+        [DataField(required: true)]
         [ViewVariables(VVAccess.ReadWrite)]
         public DamageSpecifier ClumsyDamage = default!;
 
+        /// <summary>
+        /// Chance between 0 and 1 that the character will bonk when
+        /// trying to climb something.
+        /// </summary>
+        [DataField]
+        public float BonkChance = 0.5f;
+
```

Comment:
Moving 1 bonk field out is pretty weird, would be better to keep it all on the same component.

Reply:
I can put it back, I suppose. Would it be alright to add a ClumsyLevel property to ClumsyComponent and like multiply ClumsyComponent.ClumsyLevel * BonkableComponent.BonkClumsyChance to input into the clumsy roll instead?

I like the idea of clumsiness not being a hard binary, so some creatures can be just a little bit clumsy.

Reply:
On second thought, I'll save that for another PR.